### ISSUE TITLE
remove use of regexp2

### ISSUE THREAD

**Issue by kardianos** [trustelem/zxcvbn#7]
Would you be willing to accept a PR that replaces the use of regexp2 with a simple parser or similar? Right now it is used for checking for repeated strings and for rough date matching, both of which seem reasonable to implement as a simple parser.

**Comment by vanackere** [comment]
Yes, as long as the behaviour is unchanged this looks like a worthwhile
change. Thanks !

Le jeu. 16 mai 2019 à 22:33, Daniel Theophanes <notifications@github.com> a
écrit :

**Comment by vanackere** [comment]
Hi @kardianos , do you still intend to implement this ?
